CI fails to build Eclipse plugins which require Java 11

With version CDT 10.0.0 and WTP 3.21.0, the corresponding Eclipse plugins require Java 11.
Note that the WTP version has not been released yet. The CDT build worked until 56cce95d19e36914b3b439b2f777fe55c8618968

The reason for the change is unknown.

On my local development machine I got with JDK 11 the following error:
> Caused by: java.lang.reflect.InvocationTargetException
        at com.diffplug.gradle.eclipserunner.JarFolderRunner.run(JarFolderRunner.java:51)
        at com.diffplug.gradle.eclipserunner.JarFolderRunnerExternalJvm$RunOutside.run(JarFolderRunnerExternalJvm.java:126)
        at com.diffplug.gradle.JavaExecable.main(JavaExecable.java:134)
Caused by: java.lang.SecurityException: class "org.osgi.service.log.LogLevel"'s signer information does not match signer information of other classes in the same package
        at org.eclipse.osgi.internal.log.EquinoxLogServices.<init>(EquinoxLogServices.java:94)
        at org.eclipse.osgi.internal.framework.EquinoxContainer.<init>(EquinoxContainer.java:78)
        at org.eclipse.osgi.launch.Equinox.<init>(Equinox.java:34)
        at org.eclipse.core.runtime.adaptor.EclipseStarter.startup(EclipseStarter.java:315)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher$Running.<init>(EquinoxLauncher.java:196)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher$Running.<init>(EquinoxLauncher.java:189)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher.open(EquinoxLauncher.java:174)
        at com.diffplug.gradle.eclipserunner.EquinoxLauncher.run(EquinoxLauncher.java:179)
        ... 3 more

So maybe the reason to use JDK 8 was just a problem with `com.diffplug.p2.asmaven`.
In my local environment I just worked around by using
> id 'com.diffplug.eclipse.resourcefilters'  version '3.26.1'

instead of 
> id 'com.diffplug.eclipse.resourcefilters'  version '3.29.1'

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -223,8 +223,6 @@ jobs:
           command: ./gradlew -Pcom.diffplug.spotless.include.ext.nop2=true :eclipse-jdt:changelogPush -Prelease=true --stacktrace
   ext_do_release_cdt:
     << : *env_gradle
-    docker:
-      - image: cimg/openjdk:8.0
     steps:
       - checkout
       - *restore_cache_wrapper
@@ -235,8 +233,6 @@ jobs:
           command: ./gradlew -Pcom.diffplug.spotless.include.ext.cdt=true :eclipse-cdt:changelogPush -Prelease=true --stacktrace
   ext_do_release_groovy:
     << : *env_gradle
-    docker:
-      - image: cimg/openjdk:8.0
     steps:
       - checkout
       - *restore_cache_wrapper
@@ -247,8 +243,6 @@ jobs:
           command: ./gradlew -Pcom.diffplug.spotless.include.ext.groovy=true :eclipse-groovy:changelogPush -Prelease=true --stacktrace
   ext_do_release_wtp:
     << : *env_gradle
-    docker:
-      - image: cimg/openjdk:8.0
     steps:
       - checkout
       - *restore_cache_wrapper

--- a/settings.gradle
+++ b/settings.gradle
@@ -5,7 +5,7 @@ pluginManagement {
 		//   5.6.0+ Tried to add format named 'freshmark' of type class @ spotless-freshmark.gradle:55
 		id 'com.diffplug.spotless'                 version '5.5.0'
 		// https://github.com/diffplug/goomph/blob/main/CHANGES.md
-		id 'com.diffplug.eclipse.resourcefilters'  version '3.29.1'
+		id 'com.diffplug.eclipse.resourcefilters'  version '3.27.0'
 		// https://plugins.gradle.org/plugin/com.gradle.plugin-publish
 		id 'com.gradle.plugin-publish'             version '0.14.0'
 		// https://github.com/gradle-nexus/publish-plugin/releases

